Antinoblium converts every other gas into antinoblium.

Pickle-Coding opened the pull request:
## About The Pull Request
Recommended for #91048 to be merged first to limit availability of the gas.

Adds a gas reaction that requires antinoblium. Causes other gases to get converted into antinoblium when above 20 Kelvin. Irradiates people upon skin contact.
## Why It's Good For The Game
Hypernoblium halts entropy from increasing. To make antinoblium the opposite, it accelerates entropy as fast as it can by converting every other gas into antinoblium. This brings a unique way for antinoblium to interact with the atmospherics system. Due to the grey goo nature of the gas, people will panic if this gets leaked for whatever reason, which can bring some good quality roleplay moments as people desparately try to prevent the spread of the gas or prevent the gas spreading into their departments.
## Changelog
:cl:
add: Antinoblium converts other gases into antinoblium when above 20 Kelvin.
balance: Firelocks close when they detect antinoblium.
balance: Antinoblium irradiates people upon skin contact.
/:cl:

Pickle-Coding committed:
Update reactions.dm

Adds /datum/gas_reaction/antinoblium_replication with its init_reqs() and react() overrides. It requires MINIMUM_MOLE_COUNT of antinoblium. Converts other gases into antinoblium.

Pickle-Coding committed:
Add defines.

Adds the ANTINOBLIUM_CONVERSION_DIVISOR define, set to 5. Used for the antinoblium_replication reaction.

Pickle-Coding committed:
Actually use the define.

Uses the ANTINOBLIUM_CONVERSION_DIVISOR define for /datum/gas_reaction/antinoblium_replication/react().

Pickle-Coding committed:
Update reaction_factors.dm

Adds init_factors() override for /datum/gas_reaction/antinoblium_replication.

Pickle-Coding committed:
Update reaction_factors.dm

Adds a missing parenthesis to factor in /datum/gas_reaction/antinoblium_replication/init_factors().

LemonInTheDark commented:
turbo I love you but I do need this to actually be tested

Pickle-Coding committed:
I tested it.

Removed unused variable. Fix temperature calculation.

LemonInTheDark commented:
I hate htis bot

Pickle-Coding commented:
I'll note that the testing isn't very thorough yet, but it compiles and works without any obvious errors or bugs. I'll do more rigorous testing tomorrow if BYOND behaves nicely.

sippykot commented:
so, this makes antinoblium into
a) pretty much an infinite, 0 effort money printer
b) unhinged grief gas you can use to completely fuck up atmos by injecting it into storage tanks and loop
🤒

chestlet commented:
this will make non-objective cascade delam impossible to achieve

GoldenGardens commented:
This is by far, the most retarded PR I've seen in forever

CCC23 commented:
the funny conversion gas seems busted release the scrunkler into the atmosphere to eat all the oxygen

ThatFunkyGuy commented:
I don't want to be negative, but it's difficult not to and other comments have already displayed that negativity. This is a completely stupid and usless PR, discouraging people from doing atmos at a fear of a mistake and whoops all station oxygen is gone. Even worse is griefers making the already easy to get gas and releasing it into distro and just completely fucking the station over with.

Tldr: thus sucks dount muerge!!!!!!!

sippykot commented:
Ok, what if... What if it only like, converted Plasma?

Loafin34 commented:
what if it only converted oxygen

CCC23 commented:
> what if it only converted oxygen

make it so it only happens at around room tempeture ie like 273 and turns off at over 1000k

Deadgebert commented:
please don't make some uncounterable grief gas, let atmos be used for good.
what is this trying to accomplish? are you trying to make an easier way to create antinob or do you just want another route for antag datum to end the round?

Pickle-Coding commented:
> so, this makes antinoblium into a) pretty much an infinite, 0 effort money printer b) unhinged grief gas you can use to completely fuck up atmos by injecting it into storage tanks and loop 🤒

a: It's a money multiplier not printer. It converts existing gases which can already generate money. It also isn't no effort because it's extremely dangerous to do so.
b: Yeah.
c: It can also be used as a heat pump. Converting room temperature plasma to antinoblium will raise its temperature to 58,630K, so you can use that to start reactions.


> this will make non-objective cascade delam impossible to achieve won't it?
> 
> also, just tested. spreads faster than firelocks can contain, once it's in the main halls it's over

It will still be possible for cascade since hypernoblium stops the reaction. I will see what I can do about the firelock issue.



> I don't want to be negative, but it's difficult not to and other comments have already displayed that negativity. This is a completely stupid and usless PR, discouraging people from doing atmos at a fear of a mistake and whoops all station oxygen is gone. Even worse is griefers making the already easy to get gas and releasing it into distro and just completely fucking the station over with.
> 
> Tldr: thus sucks dount muerge!!!!!!!

How are you going to accidentally create antinoblium? Seems highly unlikely.



> please don't make some uncounterable grief gas, let atmos be used for good. what is this trying to accomplish? are you trying to make an easier way to create antinob or do you just want another route for antag datum to end the round?
> 
> the suggestions to make it work on only nitrogen and oxygen don't solve anything either. this still just ends the round. you cannot reliably completely drain an area of gas, it takes way too much time. the suggestion to work only with plasma will ruin atmos cascade, you will not be able to maintain the ratio
> 
> in a 6L tank, 20.1K, 5 hnob, 20 antinob, rest plasma, this enables 3 dev internal caps (internals that explode when you breath)
> 
> I genuinely don't see a real use outside of an avenue for antagging. please reconsider.

Counterpoint: Let atmos be used for bad. Also you can already create a maxcap from internals that get triggered by breathing without needing this change, even with 6L tanks I think.

SmArtKar commented:
How quickly does it spread in practice on a map like Delta or Nebula (for multi-z purposes)? I feel like this will be A) extremely laggy and B) extremely, extremely griefy

Pickle-Coding committed:
Balancing.

The ANTINOBLIUM_CONVERSION_DIVISOR now divides the antinoblium moles instead of the not antinoblium moles for reaction_rate calculation. Makes it much slower when there is a low amount of antinoblium and faster when there is a high amount.

Sets ANTINOBLIUM_CONVERSION_DIVISOR to 90.

Sets the antinoblium requirement to MINIMUM_MOLE_COUNT * ANTINOBLIUM_CONVERSION_DIVISOR so it doesn't trigger when it doesn't have enough to actually do the reaction.

Add an early return if there isn't enough antinoblium to satisfy reaction requirements.

Includes minimum requirement change in reaction_factors.dm.

Firelocks also trigger if it detects any amount of antinoblium.

Krail commented:
Man can't wait for someone  release this gas across all ice box on high pop, so atmos just shits itself and server starts to kneel.

OverwatchVoice commented:
I don't hate the idea, a strange-matter-type gas would be interesting, but it shouldn't be piss easy to create in 5 minutes.

DaxDupont commented:
is this a joke pr?

ThatFunkyGuy commented:
>  
> How are you going to accidentally create antinoblium? Seems highly unlikely.
> 

Never said anything about making it accidentally. More of a newbie atmos will accidentally release it after making it, in any of the hundred ways canisters or chambers get destroyed. Someying like a plasma fire for example, every new atmosian causes one and they are annoying and can be delt with since they die quickly unlike what youre proposing with tha gas duplicating constantly. But this really just seems like you've never touched atmos and never made any antinoblium to think this would be a good addition at all.

Pickle-Coding commented:
> Man can't wait for someone release this gas across all ice box on high pop, so atmos just shits itself and server starts to kneel. This is just adding strain to atmos, for what?

The reaction is too slow to overwhelm planetary atmos. I tested this by releasing a million moles outside in icebox. It dissipated very quickly then settled into normal planetary air.

> I don't hate the idea, a strange-matter-type gas would be interesting, but it shouldn't be piss easy to create in 5 minutes.

I hate water I hate reagents.

> is this a joke pr? additionally if you pipe this into the distribution loop it's over for the station.

That applies to a lot of gases actually. How many times did the shuttle get called over plasma in distro?

> What if through the mass of entropy, every tick inside a pipe it has a 1% chance to melt it and release the gas

This might be something to do in the future but only if pipes get shielding like canisters do so it's not impossible pump the gas or anything.

> > How are you going to accidentally create antinoblium? Seems highly unlikely.
> 
> Never said anything about making it accidentally. More of a newbie atmos will accidentally release it after making it, in any of the hundred ways canisters or chambers get destroyed. Someying like a plasma fire for example, every new atmosian causes one and they are annoying and can be delt with since they die quickly unlike what youre proposing with tha gas duplicating constantly. But this really just seems like you've never touched atmos and never made any antinoblium to think this would be a good addition at all.

I think by the point someone is deciding to make antinoblium, they probably won't be afraid of atmospherics anymore. Antinoblium is either used to make supermatter crystals and fancy statues, or the most powerful and dangerous HFR fuel.

iamcooldan commented:
yes, but you can't make stuff with antinob anymore with the crystallizer cause it would just convert your other gasses. I'm with the torch and pitchfork crowd on this one

Pickle-Coding committed:
Add a minimum mole count requirement for firelock detection.

Adds a minimum mole requirement for firelock antinoblium detection, set to MINIMUM_MOLE_COUNT. Should make it easier to fix the firelocks using scrubbers in case they start sleeping or something.

Pickle-Coding commented:
> yes, but you can't make stuff with antinob anymore with the crystallizer cause it would just convert your other gasses. I'm with the torch and pitchfork crowd on this one

Put hypernoblium in the pipe then.

Deadgebert commented:
> Counterpoint: Let atmos be used for bad. Also you can already create a maxcap from internals that get triggered by breathing without needing this change, even with 6L tanks I think.

devastation boom in 6L can only be created using a bug that currently has a fix up.
this also breaks the creation of zaukerite. the temperature for the reaction is too low, hypernob doesn't prevent reactions until 20K

Pickle-Coding committed:
Minimum temperature requirement.

Adds a define called REACTION_OPPRESSION_MIN_TEMP, set to 20. Uses that instead of 20 for the temperature check in hypernoblium reaction oppression.

Adds a minimum temperature requirement for antinoblium replication. It's set to REACTION_OPPRESSION_MIN_TEMP.

Adds the minimum temperature requirement information to reaction_factors.dm for antinoblium replication.

GremlinSeeker commented:
I don't think this is a bad idea per-say but I think there needs to be something else to it besides just the smallest amount of antinob instantly ending the round if its used anywhere though. A mole/heat/kpa requirement anything tbh

ATH1909 commented:
your opening post does not actually explain why adding a grey goo gas that makes large regions of the station uninhabitable at the slightest hint of exposure is good for the game

can you name a single positive play pattern that this will create (and that plasmaflooding doesn't do in a way that's more interesting for the crew)?

ATH1909 commented:
(and if your only answer is "a traitor could kill everyone with it", I think a plasmaflood accomplishes that in a way that's much cooler)

ATH1909 commented:
also if it doesn't already exist you should probably add some kind of admin tool for purging all instances of antinobilium from all atmos tiles

because I genuinely don't know how else an admin is supposed to be able to (quickly) undo the damage caused by a griefer's new alt speedrunning making antinobilium and spreading it around the station, this shit is gonna be like real-life glitter

norsvenska commented:
> some kind of admin tool for purging all instances of antinobilium from all atmos tiles

fix-air verb already exists

Pickle-Coding commented:
> I don't think this is a bad idea per-say but I think there needs to be something else to it besides just the smallest amount of antinob instantly ending the round if its used anywhere though. A mole/heat/kpa requirement anything tbh
> 
> edit: didnt read your last message but thats the right direction your going in though 20kelvin isn't really fitting in with the opposite of hypernob approach tbh and much too easy

Right now it has a 0.9 mole requirement. It's quite small but not trace amounts small. It's high enough that it's possible to open a firelock and run through it without leaking enough to reach critical density in a room, though it's quite risky.

> your opening post does not actually explain why adding a grey goo gas that makes large regions of the station near-permanently uninhabitable at the slightest hint of exposure is good for the game
> 
> can you name a single positive play experience that this addition could create?

People will think antinoblium is a spooky scary gas, which can improve antinoblium RP when handling it.

LemonInTheDark commented on code/game/machinery/doors/firedoor.dm:
```diff
@@ -285,6 +285,8 @@
 
 	if(environment.temperature >= FIRE_MINIMUM_TEMPERATURE_TO_EXIST)
 		return FIRELOCK_ALARM_TYPE_HOT
+	if(environment.gases[/datum/gas/antinoblium] && environment.gases[/datum/gas/antinoblium][MOLES] > MINIMUM_MOLE_COUNT)
+		return FIRELOCK_ALARM_TYPE_HOT
```
this is kinda hacky ngl

Pickle-Coding replied:
It is. I think I detected some strange issue where the firelock would rapidly open and close when I tried to make it do its own thing. I'll investigate that later.

LemonInTheDark replied:
it's likely heat spiking and then spreading out. I can't recall if we have a grace window between high and low but maybe we should

LemonInTheDark commented on code/modules/atmospherics/gasmixtures/reactions.dm:
```diff
@@ -1161,4 +1161,49 @@
+		if(id == /datum/gas/antinoblium)
+			continue
+		var/list/gas = cached_gases[id]
+		if(. == NO_REACTION) // Let the gases get properly cleared while avoiding potential division by 0.
+			gas[MOLES] = 0
+			continue
```
when there isn't enough anti to react, we wipe all the other gases out? s weird

Pickle-Coding replied:
There is an early return earlier that prevents that. I should probably make the comments more clear on what's going on.

LemonInTheDark commented on code/modules/atmospherics/gasmixtures/reactions.dm:
```diff
@@ -1161,4 +1161,49 @@
+			gas[MOLES] = 0
+			continue
+		gas[MOLES] -= reaction_rate * gas[MOLES] / total_not_antinoblium_moles
+	antinoblium[MOLES] += reaction_rate
```
shouldn't this multiply against total_not... or is that intentional

Pickle-Coding replied:
Intentional. It's the reaction_rate divided by the composition that gas makes up of the mixture.

LemonInTheDark reviewed (changes requested):
I can get down with making this gas grey goopy (tho I think you should have like, argued for that in the pr body).

The upper ends of atmos feel kind of overcomplicated and niche, sort of due to the constraints of how they were created. I'm fine with a new creepy gas behavior existing, especially if it's this hard. It'll make for a few really cool rounds (disasters form the best roleplay), and if people are cringe admins can kill them.

Pickle-Coding committed:
Add comment.

Adds a comment to clarify an early return.

timothymtorres commented:
This is one of those ideas that sounds fun on paper but the actual effects would make the game unplayable since this becomes an easy grief tool that is worse than plasma fires.

thetaro commented:
why does this not have the balance tag actually, this is absolutely a balance change

Pickle-Coding commented:
> This is one of those ideas that sounds fun on paper but the actual effects would make the game unplayable since this becomes an easy grief tool that is worse than plasma fires.

With #91048 it shouldn't be too easy. The effort it takes to obtain it with that PR is also similar to other mass destruction things you can do. I doubt it will be spammed every round.

KingkumaArt commented:
> > This is one of those ideas that sounds fun on paper but the actual effects would make the game unplayable since this becomes an easy grief tool that is worse than plasma fires.
> 
> With #91048 it shouldn't be too easy. The effort it takes to obtain it with that PR is also similar to other mass destruction things you can do. I doubt it will be spammed every round.

But you do realize that... antinob is used for stuff other than this as it stands, right? You're making those things harder to acheive and if they screw up with it, well great they made one mistake and ruined the round for everyone else, by accident. Atmos is already very prone to causing mistakes, and if my single error literally killed everyone on the map i dont think i'd want to play atmos anymore.

NebulaFale commented:
Someone could just break into the N2/O2 tank and open an antinob can to contaminate the ENTIRE distro if this PR passes.

Pickle-Coding commented:
> > > ...
> > 
> > 
> > ...
> 
> But you do realize that... antinob is used for stuff other than this as it stands, right? You're making those things harder to acheive and if they screw up with it, well great they made one mistake and ruined the round for everyone else, by accident. Atmos is already very prone to causing mistakes, and if my single error literally killed everyone on the map i dont think i'd want to play atmos anymore.

Lets look at the current use cases of antinoblium.
- Creating supermatter shards - This is already extremely dangerous. You could accidentally dust yourself then have activated supermatter shards roaming around.
- HFR fuel - Extremely dangerous. It is by far the hardest fuel to handle, and if it blows up, it will disrupt the round harder than leaking antinoblium would.
- Zaukerite - This is used for making statues, atmos gas masks and showing off.
- Most powerful reactionless bomb - Showing off. You don't need to do this to complete any experiments, as they can be done with weaker bombs.
- Cascade delamination - Extremely dangerous. This is meant for ending the round.
- Supermatter fuel - Extremely dangerous. Supermatter shit.

All of these are already either extremely dangerous or for showing off. It is a bigger show off if what you're doing is dangerous. Risking a grey goo scenario to make a statue sounds funny. Rounds are meant to end at some point in some way so it isn't inherently a bad thing, as long as it isn't too common and the method of it ending isn't boring.

> Someone could just break into the N2/O2 tank and open an antinob can to contaminate the ENTIRE distro if this PR passes.

People can fireaxe the gas tanks and all the gases will get spaced. They can also mix the oxygen and plasma tanks, set it on fire and connect that to distro. That is just the nature of entropy of highly pressurised tanks of gas surrounded in space. Any form of sabotage to them is always going to be extremely hard to fix.

MrMelbert commented:
The "Why it's good for the game" doesn't really explain why it's good for the game, it just explains the in-universe logic behind it

So like... why is this good for the game?

timothymtorres commented:
This is basically the gas version of Cascade delamination. You have to jump through a lot of hoops to achieve that effect normally and the crew gets an ample warning and time to counter it.

Pickle-Coding commented:
> The "Why it's good for the game" doesn't really explain why it's good for the game, it just explains the in-universe logic behind it
> 
> So like... why is this good for the game?

I have updated the Why It's Good For The Game section.

> This is basically the gas version of Cascade delamination. You have to jump through a lot of hoops to achieve that effect normally and the crew gets an ample warning and time to counter it.

The cascade can't be stopped once it's released and instantly dusts everything. Antinoblium can be blocked off, only destroys gases and heats up the room.

Kiromaka commented:
Cascade getting practically removed from the game for... uh... funny grief gas ?  

I pray on maintainers to look at PR reactions before deciding to merge 🙏🏻

Pickle-Coding committed:
Set reaction results.

Apply the SET_REACTION_RESULTS() macro for antinoblium_replication/react().

Pickle-Coding committed:
Merge branch 'master' of https://github.com/tgstation/tgstation into grey-goo

moocowswag commented:
Is even the smallest amount of antinoblium visible? I would like for people to be able to see and run from the cloud while seeing it encroach and corner people.

Pickle-Coding commented:
> Is even the smallest amount of antinoblium visible? I would like for people to be able to see and run from the cloud while it encroaches and corners people.

0.25 moles is required to be visible. It only replicates when there are at least 0.9 moles.

Pickle-Coding committed:
Minimum mole requirement change.

Changes the minimum mole requirement of antinoblium_replication to MOLES_GAS_VISIBLE. The reaction_rate of antinoblium_replication/react() will always react at at least MINIMUM_MOLE_COUNT if there are enough total_not_antinoblium_moles to do so.

Removes redundant checks for antinoblium_replication/react().

Pickle-Coding commented:
Changed the minimum mole requirement to 0.25 moles, which is the visible moles requirement. I think it would look weird if there was a visible amount of antinoblium but it doesn't do its thing.

moocowswag commented:
Thats gonna make it spread alot more :(

Pickle-Coding commented:
> Thats gonna make it spread alot more :(

It would, but it still takes time for a room to build up antinoblium for it to leak enough to contaminate another room when opening a firelock. 0.9 was only picked as a side effect of how I did the reaction with minimum mole checks for the reaction rate.

Pickle-Coding committed:
A bit more forgiving.

Removes the MINIMUM_MOLE_COUNT as the minimum reaction rate and instead just check for total_not_antinoblium moles to clear the gases.

moocowswag commented:
I went ahead and tested the commit before the latest one, once it gets into a room you are fine for nearly 2 minutes

After it reaches 60% of the total mole count (when itl still be only 400ish kelvin) it will snowball and go up 2% every second while heating up the room dramatically, ending at over 1,300 kelvin

after about 3-4 minutes the room naturally cools down and you can safely just go back in there with internals on.

Pickle-Coding commented:
I was under the assumption that being in a 373.15K room would burn your skin. It shows the too hot indicator but doesn't actually do damage. Any ideas for it to maintain relevance once the room cools down to that point?

Pickle-Coding committed:
Antinoblium irradiates shit.

Adds the /datum/species/handle_gas_interaction() proc. Used for handling gas interactions on skin exposure. Called in /datum/species/proc/handle_environment.

Adds an antinoblium skin interaction that irradiates the target.

Pickle-Coding commented:
Ideally antinoblium removing the oxygen of the station would be the main threat, but with the current state of gas tanks, that is not a real problem. I'm going to make it irradiate on skin exposure for now.

moocowswag commented:
> I was under the assumption that being in a 373.15K room would burn your skin. 

Not even lizards take damage in that temperature,

temp stabilization is able to overpower high temp damage well into over 400k because of the change to having internal heat that trends towards surface heat in carbons



This is why lizards secretly are not good against heat and it showing up as one of their species beneficial traits should probably be removed because its noob bait that actually gets players killed (its only ethereal that are good against heat)

A humans temp stabilization is so strong and infinite that by the time they go into crit from 500 kelvin, a lizard in the same room as them will already be dead

Unit2E commented:
Gonna throw my thoughts out there: I like it.

The worst impact this change could have is that it is finally a better flood than plasma is, and create accidents that may or may not be fun.

I think this adds a new hazard which might be fun to work around and create some cool feeling dire scenarios should it be spread, while not being TOO high danger. I've largely always held the opinion that, because of a lot of practical reasons, plasma floods have always been the strongest one. Tritium and hydrogen burn through floors and there's a limit to temp damage anyway, CO2 and N2O are countered by items literally everybody spawns with (you can make them cold I guess?). Freon is countered by winter coats, extinguishers, or just scrubbing it... And that's basically it for dangerous gases to flood.

Not only that, but several of those are either more limited in quantity, or take time and effort to make. Meanwhile it's trivially easy to put 100,000s of moles of plasma in a can, for which there are several genuine reasons you'd be doing this, and hook it up into distro. Round over. I truly do not believe that no gas should ever compete with how effective of a flood plasma is, especially if there is effort required and the gas has somewhat limit uses anyway.

Lastly, only pain point: the HFR makes a lot of this stuff, and is very inconspicuous to run. Maybe problematic, maybe not, eh.

Pickle-Coding commented:
> Lastly, only pain point: the HFR makes a lot of this stuff, and is very inconspicuous to run. Maybe problematic, maybe not, eh.

The HFR is big and can be heard outside of engineering if it's running at a high enough power to generate antinoblium. It also consumes a lot of power, and releases zaps when creating antinoblium. I don't think that's inconspicuous, though the HFR rooms are kinda isolated from the rest of the station in some maps. I could limit antinoblium production to power 6 or make it release anomalies if it ends up being too easy or gameable, but I'll leave that for a future PR after seeing how this turns out.

Unit2E commented:
Bad choice of words on my part. What I mean is that the HFR, while loud, will very often be run for non-antag / not antinoblium reasons. It provides quite a lot of cover compared to 5000 MeV SM running with hyper noblium present in the Tesla coils.

Anyway, I don't think it needs changing, just an observation! Depending on how it turns out, I think limiting it to level 6 / anomalies would help. Regardless, excited to see how this plays out!

Pickle-Coding commented:
I'm fine if people don't identify it as antagonistic. I just prefer the method being loud and wacky, whether people know what's going on or not.

LemonInTheDark commented:
I like it and turbo seems happy, we vibe

Tostah commented:
the reasoning as to why this is good for the game is lackluster at best. This is ill advised, potentially game breaking. with the 69 comments and nobody thought 'wow maybe this isnt a good idea'

LemonInTheDark commented:
I considered it and then decided I liked it instead.

money thing is an issue yeah, I believe we have falloff on credits, might need to lower it though (doesn't make sense for more then the first bit to be expensive because the user can just dupe it for free)

the conversion issue was discussed in #91048, you do not fully understand our reaction system. hypernob supression superceeds all other possible reactions.

Pickle-Coding commented:
If economy thing becomes an issue then we could make it so centcom has a chance to accidentally leak antinoblium after selling it to them (they have no idea what it does), then they no longer buy it from you while centcom is full of antinoblium.

Tostah commented:
> I considered it and then decided I liked it instead.
> 
> money thing is an issue yeah, I believe we have falloff on credits, might need to lower it though (doesn't make sense for more then the first bit to be expensive because the user can just dupe it for free)
> 
> the conversion issue was discussed in #91048, you do not fully understand our reaction system. hypernob supression superceeds all other possible reactions.

if anti nob turns any gas besides hyper nob into anti nob, then any waste the SM produces gets turned into anti nob. I can see this making cascades physically impossible as the the mix will always go out of range.

LemonInTheDark commented:
please read the things I have written on that pr you do not understand how hypernobilium works

Tostah commented:
> please read the things I have written on that pr you do not understand how hypernobilium works

I also strongly dislike the other PR you mentioned. I dunno how two PRs that are receiving RESOUNDING negative attention got merged in the first place. 

But I get it. Hyper nob stops all reactions, fine. So the cascade isn’t affected. 

This combined with this other PR means that you should just produce like 1 mole of anti nob then stick it in a chamber with 99999 moles of other gasses. To then produce infinite anti nob. 

This goes BEYOND being a normal flood. This is game-destroying shit. You want to remove every single gas from atmos so nobody can do anything? Add a very small amount of anti nob using the gas filters in distribution. 

You can recover from a plasma flood. It would be very difficult to recover if every single gas in the entire station was replaced with anti  nob

sippykot commented:
tg will just merge any slop even if every player hates it, its always been true
the only way to force course-correction is boycott but no one is going to do that over dumb gas changes because they dont affect the game enough, like wallening did

GoldenGardens commented:
Ya'll are retarded for merging this slop.
